test/templates/api/repositories/template-preview-unknown-variable/002-poll/expected-response.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "template.preview/%s",
+  "type": "template.preview",
+  "url": "https://templates.keboola.local/v1/tasks/template.preview/%s",
+  "status": "error",
+  "isFinished": true,
+  "createdAt": "%s",
+  "finishedAt": "%s",
+  "duration": %d,
+  "error": "%s Unknown variable: InstanceIdShort"
```

Comment:
Is this variable or function. We say function but I dont know exactly jsonnet specification. So can you answer this not just resolve it?

Reply:
> Is the InstanceIdShort variable of function? Can you check jsonnet whether the functions and variables are different things? In case it is the same just resolve the comment.

variable in jsonnet:
local InstanceIdShort = "abcd1234";  // Variable

local InstanceIdShortFunc(id) = id[:4];  // Function

{
  variableUsage: InstanceIdShort,         // Uses variable: "abcd1234"
  functionUsage: InstanceIdShortFunc(InstanceIdShort),  // Calls function: "abcd"
}

Our [function](https://github.com/keboola/keboola-as-code/blob/07e51915cac1f3d72399849d7c36e3489104b569/internal/pkg/template/context/use/context.go#L229) registry (create) variables (functions). Here are [tests](https://github.com/keboola/keboola-as-code/blob/f30553818f4379bbf8dd19dd46b2a693958fc8d1/internal/pkg/encoding/jsonnet/vm_test.go#L12).


> In case it is the same just resolve the comment.

and YES I have resolved it, beacuse its the same.

Reply:
thanks